get-started.md

```diff
@@ -1,5 +1,69 @@
 ---
 layout: get-started
-title: Get Started
+title: Kogito - Get Started
 permalink: /get-started/
 ---
+
+<div class="grid__item width-4-12 hide-mobile toc" markdown="1">
+## Table of Contents
+### Getting Started
+
+ - [Kogito business automation overview](https://docs.jboss.org/kogito/release/latest/html_single/#con_kogito-automation)
+ - [Creating and running your first Kogito services](https://docs.jboss.org/kogito/release/latest/html_single/#chap_kogito-creating-running)
+ - [Example applications with Kogito services](https://docs.jboss.org/kogito/release/latest/html_single/#ref_kogito-app-examples_kogito-creating-running)
+ - [Deploying Kogito services on OpenShift](https://docs.jboss.org/kogito/release/latest/html_single/#chap_kogito-deploying-on-openshift)
+ - [Travel agency tutorial for Kogito services on OpenShift](https://docs.jboss.org/kogito/release/latest/html_single/#con_kogito-travel-agency)
+
+### Guides
+
+ - [Get More Guides]({{site.baseurl}}/guides)
+
+</div>
+<div class="grid__item width-8-12 width-12-12-m gs-content">
+<div class="guide-item" markdown="1">
+## Kogito business automation overview
+
+Learn what is Kogito and when it can become an essential core of your domain-specific cloud-native set of services.
+
+<a href="https://docs.jboss.org/kogito/release/latest/html_single/#con_kogito-automation" class="button-cta secondary">READ THE GUIDE</a>
+</div>
+
+<div class="guide-item" markdown="1">
+## Creating and running your first Kogito services
+
+As a developer of business processes and rules, you can use Kogito business automation to build cloud-native applications that adapt to your business domain and tooling.
```

Comment:
not sure if this phrase is ok here, at least for me the "applications that adapt to your business and tooling" is not clear.
, can you please clarify?

Reply:
Adding @sterobin to comment

Reply:
IMHO we could improve it, basically you have only two available tooling (maybe 3 with the chrome plugin), bpmn|dmn.new or the vscode plugin, if a developer uses eclipse or intelliJ he will have to adapt their development tools to design kogito services, does it make sense?

Reply:
@spolti , yep, makes sense. Though apparently others disagree since that was the sanctioned messaging, haha! I saw your feedback on that other PR. Thank you! Feel free to bring this up as well.